### ISSUE TITLE
Documentation to JSON_LISTINGS_FEED

### ISSUE THREAD

**Issue by KemnaMOL** [jlevers/selling-partner-api#820]
## Problem description:
Hello,
I can´t find a documentation who I can upload data with the newer JSON_LISTINGS_FEED.
At the moment I use the older XML-version.

I don´t want to use the new Transformer.

**Comment by jlevers** [comment]
There's pretty thorough documentation on migrating to the Listings API (the JSON_LISTINGS_FEED is basically just a wrapper for sending many Listings API requests at the same time) [here](https://developer-docs.amazon.com/sp-api/docs/listings-management-workflow-migration). Once you've built the feed contents, the process is the same as submitting any other feed type, which is documented in the README, [here](https://github.com/jlevers/selling-partner-api#uploading-a-feed-document).

Out of curiosity, what's holding you back from using the feed transformer?

**Comment by KemnaMOL** [comment]
Many thanks. 
I think it´s easyer to give the right data, than installing a new tool. Also I can mix price and  stock data at one time.
Best regards, René

**Comment by KemnaMOL** [comment]
Your documentation is only for XML-files. What I have to change?
feedType  to JSON_LISTINGS_FEED and at feedContents the JSON-file is enough?

**Comment by jlevers** [comment]
Yeah, all you need to do is change the feed type and the library will handle the rest.

**Comment by KemnaMOL** [comment]
Yeah, I got it in this way.
But nothing changed. Now I will try to read the feed status and answer.

**Comment by NoxArt** [comment]
Check out this https://developer-docs.amazon.com/sp-api/docs/listings-management-workflow-migration

Also you can use this to get JSON feed schema

> $schema = $definitionsApi->getDefinitionsProductType($type, [ $marketplace->getId() ], $sellerId, locale: 'en_US');		> $meta = $schema->getSchema()->getLink()->getResource();

base $type is "PRODUCT"

Here are some examples + a place where to ask questions
https://github.com/amzn/selling-partner-api-models/tree/main/schemas/feeds

**Comment by NoxArt** [comment]
Also note that JSON has way harsher restrictions on feed file size and message count - aside from format conversion we also had to implement way to generate the feed into multiple files + internal queue that is sending the files one by one with some delay (and retry)

**Comment by NoxArt** [comment]
If you have a question feel free to contact me @KemnaMOL

**Comment by jordan26** [comment]
Hi @NoxArt - mind me getting in touch, I am really struggling to understand the process of converting what I had for sending stock over to the JSON formatting. I've spent weeks now reading 'snippets' of code but I need a full overview realistically. See my latest issue thread for example: https://github.com/jlevers/selling-partner-api/issues/829

Happy to discuss over email or other means if easier.

**Comment by jordan26** [comment]
> Yeah, I got it in this way. But nothing changed. Now I will try to read the feed status and answer.

@KemnaMOL - I am providing my completed code here for when I finish getting JSON working too. What I have currently would work to send JSON too, I just need to find out which format for my use case.

https://github.com/jlevers/selling-partner-api/issues/829
